components/cardVideo/cardVideo.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+// Components
+import ScoreStarSVG from '../svg/scoreStarSVG';
+import IconPlaySVG from '../svg/iconPlaySVG';
+
+const CardVideo = ({ card }) => {
```

Comment:
I saw you are just using the title property on this card, so probably we can change the props instead of passing a `card` object you can pass directly the title:

```jsx
<CardVideo title="Hello World!" />
```

Reply:
Thank you so much, I think that we need iterate for each title or something property more that we could need after. I have simulated the array videos that are in the home like the title that we are access to the course title, for example, `card.title = course 1, course 2, course 3, etc.` what do you think about this?

Reply:
Yes, in that case the prop received will be an array as you mentioned but each element on that array can be from the `iCard` interface, but since this component is related to a single video card, it would be better if it only receives the props for the card defined on that interface

Reply:
Sure, I have destructured card `const CardVideo = ({ card: { title } }) => {....`  thank you for your suggestion c: